src/containers/About/About.js

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { Table } from '@tektoncd/dashboard-components';
+import { InlineNotification } from 'carbon-components-react';
+import { getInstallProperties } from '../../api';
+import { isWebSocketConnected } from '../../reducers';
+
+const initialState = {
+  dashboardInfo: [],
+  isLoaded: false,
+  isNotFinished: true,
+  error: ''
+};
+
+export /* istanbul ignore next */ class About extends Component {
+  constructor(props) {
+    super(props);
+    this.state = initialState;
+  }
+
+  componentDidMount() {
+    this.fetchDashboardInfo();
+  }
+
+  setErrorState(errorsFound) {
+    let errorMessage = 'Error getting version of ';
```

Comment:
we need to use react-intl for user-facing strings, also can't use concatenation to build these strings due to ordering / syntax differences in many languages

for building the list we can't manually build comma-separated strings, instead we should use react-intl's formatted list support, see https://github.com/formatjs/react-intl/blob/master/docs/Components.md#formattedlist